src/main/scala/com/databricks/spark/sql/perf/tpcds/SimpleQueries.scala

```diff
@@ -138,7 +138,38 @@ trait SimpleQueries extends Benchmark {
               |  i_item_id
               |limit 100
               |-- end query 1 in stream 0 using template query7.tpl
-            """.stripMargin)
+            """.stripMargin),
+            
+      ("q7-bigtable-selfjoin-1",   """
```

Comment:
Since it is not very related to q7, I'd change the name to "store_sales-selfjoin-1".